src/util/LinkButton.js

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import {withRouter} from 'react-router';
 
+const StyledButton = styled.button`
```

Comment:
Instead of styling the button in this class, you should style the imported LinkButton in the file you'll be using it. This way, we can have Link buttons styled differently in each file that its used

Reply:
Styled(componentName) i.e. to override a style for a preexisting component, I'll look at the documentation